advanced_functionality/autogluon-tabular/AutoGluon_Tabular_SageMaker.ipynb

```diff
@@ -595,6 +603,165 @@
    "source": [
     "predictor.delete_endpoint()"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Explainability with Amazon SageMaker Clarify\n",
+    "\n",
+    "There are expanding business needs and legislative regulations that require explainations of why a model mades the decision it did. SHAP (SHapley Additive exPlanations) is an approach to explain the output of machine learning models. SHAP values represent a feature's contribution to a change in the model output. SageMaker Clarify uses SHAP to explain the contribution that each input feature makes to the final decision."
```

Comment:
"represent a feature's contribution to a change in the model output"
->
"quantify each feature's contribution to a change in the model's  output from some baseline reference value."

Reply:
Also if you end up adding the permutation feature importance section I described above, then it would be good to clarify here:

"Unlike permutation feature importance scores which quantify how much each feature contributes to a trained predictor's  accuracy (over many datapoints), SHAP values quantify how much each feature affects particular model predictions (for individual datapoints)."